tests/model_serving/model_runtime/vllm_omni/utils.py

```
@@ -0,0 +1,291 @@
+        },
+        verify=False,
+        timeout=timeout,
+    )
+    validate_tts_output(response=response, response_format=response_format)
+    return response
+
+
+def discover_tts_voice(isvc: InferenceService, preferred: str = "vivian") -> str:
+    """Discover available TTS voices from the /v1/audio/voices endpoint."""
+    url = get_exposed_isvc_url(isvc=isvc)
+    resp = requests.get(f"{url}/v1/audio/voices", verify=False, timeout=30)
```

Ruff (0.16.1)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)